addons/core/addon/helpers/truncate-list.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+/**
+ * Use this helper to truncate the list if there are more items than the limit specified
+ *
+ * @example
+ *
+ * {{truncate-list array limit}}
+ */
+
+import Helper from '@ember/component/helper';
+import { inject as service } from '@ember/service';
+
+export default class extends Helper {
+  // =services
+  @service intl;
+
+  // =compute method
+
+  /**
+   * This helper truncates the list if there are more items than the limit specified
+   * and returns the original list appended with +N remaining items
+   * @param {array} list List of items
+   * @return string
+   */
+
+  compute([list, limit = 3]) {
+    if (list) {
+      const numberOfItems = Object.keys(list).length;
+      const items = list.map(({ value }) => value);
+      if (numberOfItems > limit) {
+        const remainingItems = items.splice(limit).length;
+        return (
+          items.join(', ') +
+          `, +${remainingItems} ${this.intl.t('actions.more')}`
```

Comment:
We should make this phrase entirely in the translation and take in a variable since some languages might have a different order between the number and counter

Reply:
That's a good point! will update

Reply:
Awesome catch!! Indeed with @ZedLi comment